utils/node_local_test.py

```diff
@@ -998,18 +1029,21 @@ def wait_for_exit(self):
         # prefix to the src dir.
         self.valgrind.convert_xml()
 
-def get_pool_list():
+def get_pool_list(server):
     """Return a list of valid pool names"""
     pools = []
 
-    for fname in os.listdir('/mnt/daos'):
-        if len(fname) != 36:
-            continue
-        try:
-            uuid.UUID(fname)
-        except ValueError:
-            continue
-        pools.append(fname)
+    for idx in range(server.engines):
+        for fname in os.listdir('/mnt/daos_{}'.format(idx)):
```

Comment:
This is sort of an ugly hack, IMO. I know it's just modifying existing code to support > 1 server, but it would be better to use `dmg system list-pools --json` to make this a bit more resilient/future-proof.

Reply:
That would be a good change, I'll aim to include it at some point.